### PR TITLE
DNA recovery surgery usable to de-husk people

### PR DESCRIPTION
**Pull request by SomeguyManperson**
:cl:  
tweak: DNA recovery can now fix husking
tweak: DNA recovery now unlocked at experimental surgery instead of advanced surgery
/:cl:

### CODE REVIEW

**Comment by Mqiib** [comment]
[something about hugbox]

File: code/modules/surgery/advanced/dna_recovery.dm [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	if(!..())
 		return FALSE
 
-	if(!HAS_TRAIT_FROM(target, TRAIT_BADDNA, CHANGELING_DRAIN))
+	if(!HAS_TRAIT_FROM(target, TRAIT_BADDNA, CHANGELING_DRAIN) || !HAS_TRAIT(target, TRAIT_HUSK))
```
Comment (alexkar598) [inline code/modules/surgery/advanced/dna_recovery.dm]: Isnt this supposed to be an AND instead of an OR

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/surgery/advanced/dna_recovery.dm (hunk 1 above)
Comment (SomeguyManperson) [reply code/modules/surgery/advanced/dna_recovery.dm]: Fuc

**Review by alexkar598 — APPROVED** [review]

**Comment by alexkar598** [comment]
Im not too sure about this but why not

**Pull request merged**